PedDetection.md

```diff
@@ -26,7 +26,7 @@ This specification is for either a thermal imaging detection system or a video d
     1. Electromagnetic Compatibility: Germane portions of FCC 47 CFR Part 15, Subpart B; pedestrian detection system manufacturer to describe further in its response.
     1. Environmental Compliance
         1. Ingress Protection (IP) rating of 67 (dust tight and submersible up to three feet depth) or better.
-        2. Operating temperature of -35°F to +165°F or better.
+        2. Operating temperature of -35°F to +165°F or better under normal operating loads.
```

Comment:
@MatthewJMcAllister to convert this to a UL and CFA rating parameter. Ask respondents to report on their operating range.